ai_eval/templates/chatbox.html

```diff
@@ -12,10 +12,10 @@
         </div>
     </div>
 
-    <div>
-        <div id="chat-history" role="log" aria-live="polite" aria-relevant="additions text" aria-atomic="false">
-            <div class="chat-message-container" id="chat-spinner-container">
-                <div class="chat-message" id="message-spinner" style="display: none;" aria-hidden="true">
+    <div id="chatbox">
```

Comment:
Can we leave this refactoring of for a follow up PR later? (this, chatbox.css, chatbox.js)